src/pushsource/_impl/model/vms.py

```diff
@@ -24,6 +24,17 @@ def __repr__(self):
         return f"{cls_name}.{self.name}"
 
 
+@attr.s()
+class VMICloudInformation(object):
```

Comment:
Could you please use "Info" rather than "Information"?

There are already classes in this library using "Info" (KojiBuildInfo, ContainerImagePullInfo) so we should maintain consistency.

Reply:
Fixed, thanks for reminding me about that!